src/WebJobs.Script/Diagnostics/FileTraceWriter.cs

```diff
@@ -131,6 +131,14 @@ public override void Trace(TraceEvent traceEvent)
                 throw new ArgumentNullException("traceEvent");
             }
 
+            object value;
+            if (traceEvent.Properties.TryGetValue(ScriptConstants.AzureFunctionsTracePropertyIsSystemTraceKey, out value) 
+                && value is bool && (bool)value)
```

Comment:
The idea was to have a way to filter out detailed system logs from file (user) logging. Still deciding whether this is needed.